airflow/providers/cncf/kubernetes/CHANGELOG.rst

```diff
@@ -25,17 +25,23 @@ Changelog
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
+* ``Parameter is_delete_operator_pod default is changed to True (#20575)``
 * ``Simplify KubernetesPodOperator (#19572)``
-* ``Delete pods by default in KubernetesPodOperator (#20575)``
 * ``Move pod_mutation_hook call from PodManager to KubernetesPodOperator (#20596)``
 * ``Rename ''PodLauncher'' to ''PodManager'' (#20576)``
 
-.. warning:: Many methods in ``KubernetesPodOperator`` and ``PodLauncher`` have been renamed.
-    If you have subclassed ``KubernetesPodOperator`` will need to update your subclass to reflect
-    the new structure. Additionally ``PodStatus`` enum has been renamed to ``PodPhase``.
+Parameter is_delete_operator_pod has new default
+````````````````````````````````````````````````
+
+Previously, the default for param ``is_delete_operator_pod`` was ``False``, which means that
+after a task runs, its pod is not deleted by the operator and remains on the
+cluster indefinitely.  With this release, we change the default to ``True``.
 
 Notes on changes KubernetesPodOperator and PodLauncher
 ``````````````````````````````````````````````````````
```

Comment:
I can't suggest it, but I think we want a newline here?

Reply:
you mean you want 2 newlines bettween `cluster indef` and `notes on changes`?

Reply:
oh after

Reply:
Between the header (``````) and the first line of the paragraph. Maybe it isn't required though 🤷‍♂️

Reply:
done

Reply:
Yep, like that 👍